jerry-core/parser/regexp/re-parser.cpp

```diff
@@ -403,11 +467,12 @@ re_parse_char_class (re_parser_ctx_t *parser_ctx_p, /**< number of classes */
         append_char_class (re_ctx_p, 0x2030UL, 0x205EUL);
         append_char_class (re_ctx_p, 0x2060UL, 0x2FFFUL);
         append_char_class (re_ctx_p, 0x3001UL, 0xFEFEUL);
```

Comment:
@LaszloLango, if I understand correctly, these ranges uniquely correspond to some range identifier (`'S'` in the case). Maybe, we shouldn't put the ranges to context, but instead put only unique range identifier, and then use the identifier during range checking. What do you think about this?